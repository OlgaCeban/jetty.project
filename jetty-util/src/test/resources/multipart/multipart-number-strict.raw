--P6Uyk-Vikfbk_NqTfVF4DwNXrIxpN451pD1
Content-Disposition: form-data; name="pi"
Content-Type: text/plain
Content-Transfer-Encoding: 8bit

3.14159265358979323846264338327950288419716939937510
--P6Uyk-Vikfbk_NqTfVF4DwNXrIxpN451pD1--
