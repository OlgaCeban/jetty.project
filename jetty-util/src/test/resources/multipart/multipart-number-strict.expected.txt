Content-Type|multipart/form-data; boundary="P6Uyk-Vikfbk_NqTfVF4DwNXrIxpN451pD1"
Parts-Count|1
Part-ContainsContents|pi|3.14159265358979323846264338327950288419716939937510
